src/test/java/org/apache/sysds/test/functions/codegenalg/partone/AlgorithmKMeans.java

```diff
@@ -39,14 +39,14 @@
 
 	//private final static double eps = 1e-5;
 	
-	private final static int rows = 2972;
-	private final static int cols = 972;
+	private final static int rows = 1241;
+	private final static int cols = 83;
 		
 	private final static double sparsity1 = 0.7; //dense
 	private final static double sparsity2 = 0.1; //sparse
 	
 	private final static double epsilon = 0.000000001;
-	private final static double maxiter = 10;
+	private final static double maxiter = 50;
```

Comment:
What is the reason for changed params? Speed?

Reply:
Kmeans arguments had to be changed to make it possible/more likely for kmeans to actually converge to find some clusters. The previous settings made this highly unlikely therefore the tests ended calling stop resulting in failing tests. This did not crash previously because calling stop would not make the tests fail.